examples/tfengine/generated/team/live/example-prod-apps/resources/main.tf

```diff
@@ -23,6 +23,39 @@ terraform {
 }
 
 
+
+module "example_instance_template" {
+  source  = "terraform-google-modules/vm/google//modules/instance_template"
+  version = "~> 3.0.0"
+
+  name_prefix        = "example-instance-template"
+  project_id         = var.project_id
+  region             = "us-central1"
+  subnetwork_project = "example-prod-networks"
+  subnetwork         = "example-instance-subnet"
+
+  service_account = {
+    email  = "${google_service_account.example_sa.email}"
+    scopes = ["cloud-platform"]
+  }
+
+  enable_shielded_vm = true
```

Comment:
This actually puts limitation on the image family. If they want to use a unsupported image family, this might fail.

Reply:
Made it overridable.